sdk/cosmos/azure-cosmos-test/src/main/java/com/azure/cosmos/test/faultinjection/FaultInjectionServerErrorResult.java

```diff
@@ -14,17 +14,20 @@ public final class FaultInjectionServerErrorResult implements IFaultInjectionRes
     private final Duration delay;
 
     private final Boolean suppressServiceRequests;
+    private final Double applyPercentage;
 
     FaultInjectionServerErrorResult(
         FaultInjectionServerErrorType serverErrorTypes,
         Integer times,
         Duration delay,
-        Boolean suppressServiceRequests) {
+        Boolean suppressServiceRequests,
+        Double applyPercentage) {
```

Comment:
NIT:  find the name applyPercentage really weird. I assume it means failureToBeAppliedPercentage or faultApplicationPercentage?

Reply:
maybe just injectionRate?